apps/web/src/pages/SearchRequests/RequestConfirmationPage/RequestConfirmationPage.tsx

```diff
@@ -0,0 +1,167 @@
+      }),
+      url: paths.search(),
+    },
+    {
+      label: intl.formatMessage({
+        defaultMessage: "Request submitted",
+        id: "0zo274",
+        description: "Link text for request confirmation breadcrumb",
+      }),
+      url: paths.requestConfirmation(requestId || ""),
+    },
+  ]);
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Successful request",
+    id: "DcpFle",
+    description: "Page title for the request confirmation page.",
+  });
+
+  return requestId ? (
+    <>
+      <SEO title={pageTitle} />
+      <Hero
+        title={pageTitle}
+        subtitle={intl.formatMessage({
+          defaultMessage: "Your request was submitted successfully.",
+          id: "rVgBGi",
+          description: "Subtitle for the request confirmation page.",
+        })}
+        crumbs={crumbs}
+      />
+      <div data-h2-container="base(center, large, x1) p-tablet(center, large, x2)">
+        <Text data-h2-font-size="base(3rem)">
+          {intl.formatMessage({
+            defaultMessage: "We got it!",
+            id: "hEa14D",
+            description:
+              "Lead in text for the page content on the request confirmation page",
+          })}
+        </Text>
+        <Text>
+          {intl.formatMessage({
+            defaultMessage:
+              "We have received your request and will review it as soon as possible.",
+            id: "lA+c8E",
+            description:
+              "Paragraph one, message to user the request was received",
+          })}
+        </Text>
+        <Text>
+          {intl.formatMessage(
+            {
+              defaultMessage:
+                "Your tracking number for this request is: <strong>{requestId}</strong>",
+              id: "amxWPg",
+              description:
+                "Message to the user about the ID of their request for referencing the request",
+            },
+            { requestId },
+          )}
+        </Text>
+        <Heading level="h2" size="h4" data-h2-font-weight="base(700)">
+          {intl.formatMessage({
+            defaultMessage: "What you can expect",
+            id: "N/Vcp3",
+            description:
+              "Heading for the section about user expectations for their request",
+          })}
+        </Heading>
+        <Text>
+          {intl.formatMessage({
+            defaultMessage:
+              "You will receive a follow up on your request within the next 2 to 4 business days.",
+            id: "ltn4Er",
+            description:
+              "Description of when the user should expect a response to their request",
+          })}
+        </Text>
+        <Text>
+          {intl.formatMessage(
+            {
+              defaultMessage:
+                "If you have not heard from us or have any immediate questions and request further guidance, please get in touch with use at: <mailLink>recruitmentimit-recrutementgiti@tbs-sct.gc.ca</mailLink>",
```

Comment:
please get in touch with us*
might need to double check the corresponding translation too

Reply:
Good eye! Fixed in [fix type on en request confirmation page](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5539/commits/d05bdf0a2c92127df6ff3600a9fc9995885c1e2a). FR looked good but maybe @mnigh could confirm?

Reply:
heh, typo in the commit message to fix a typo 😓

Reply:
passes the google check at least

![image](https://user-images.githubusercontent.com/40485260/215880449-fd61df35-7a61-446e-8e41-fe0279572d82.png)

Reply:
I think Google Translate autocorrects and hides the errors. It does not pass my test! 😆 

At the very least, the part _de nos Nouvelles_ seems wrong, especially the capital N. The sentence in English is also awkward, I would recommend having it re-written and then translated again in French.